country-a-service/epps-service/src/main/java/dk/nsp/epps/service/exception/DataRequirementException.java

```diff
@@ -0,0 +1,11 @@
+package dk.nsp.epps.service.exception;
+
+public class DataRequirementException extends RuntimeException {
```

Comment:
It is kind of unwieldy. All of our lambda statements or streams will need explicit handling of these exceptions, because they are technically "anonymous classes". It seems to be a known problem: https://www.baeldung.com/java-lambda-exceptions

I think that having these two "supposed to bubble up to the controller" exceptions as unchecked is a good idea - and then have everything else as checked.